### PR TITLE
Port Crashlytics quickstart to tvOS

### PR DESCRIPTION
**Pull request by judyliu14**

### CODE REVIEW

**Comment by judyliu14** [comment]
Note: Many file changes were resulted from deleting files in an extra Shared folder since they appear to not be significant. The current crash button on tvOS is triggered by pressing the "enter" key when simulating on tvOS

**Comment by paulb777** [comment]
Please investigate the CI failures

**Comment by morganchen12** [comment]
Once the current CI passes, please add a CI step for building the tvOS target as well.

File: .github/workflows/crashlytics.yml [hunk 1]
```diff
@@ -63,3 +63,15 @@ jobs:
...
           SAMPLE: Crashlytics
+      - name: Build SwiftUI (tvOS)
+        run: |
+          cd crashlytics
+          ../scripts/install_prereqs/crashlytics.sh
+          xcodebuild \
+          -project CrashlyticsExample.xcodeproj \
+          -scheme 'CrashlyticsSwiftUIExample' \
+          -sdk appletvsimulator \
+          -destination 'platform=tvOS Simulator,name=Apple TV' \
```
Comment (morganchen12) [inline .github/workflows/crashlytics.yml:74]: You'll need to change `name` to `Any tvOS Simulator Device`.

**Review by morganchen12 — APPROVED** [review]
LGTM on green CI.

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -385,6 +479,74 @@
...
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_STYLE = Automatic;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_ASSET_PATHS = "\"CrashlyticsSwiftUIExample/Preview Content\"";
```
Comment (morganchen12) [inline crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj:496]: Looks like this path is causing CI to fail. Try removing it (via Xcode or manually).

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -385,6 +479,74 @@
...
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_STYLE = Automatic;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				DEVELOPMENT_ASSET_PATHS = "\"CrashlyticsSwiftUIExample/Preview Content\"";
```
Comment (morganchen12) [inline crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj:532]: If you choose to remove the path manually, do so here as well.

**Review by morganchen12 — APPROVED** [review]

**Review by morganchen12 — APPROVED** [review]
LGTM

**Review by paulb777 — APPROVED** [review]

**Pull request merged**